tests/suites/test_suite_pkcs11_client.data

```diff
@@ -5,3 +5,11 @@ pk_import_sign:"data_files/server3.key"
 PKCS#11 ECDSA generate and sign
 depends_on:MBEDTLS_PK_C:MBEDTLS_ECDSA_C
 pk_generate_sign:MBEDTLS_PK_ECDSA
+
+PKCS#11 ECDSA generate, sign and verify with Cryptoki
+depends_on:MBEDTLS_PK_C:MBEDTLS_ECDSA_C
```

Comment:
This should also depend on `MBEDTLS_ECP_DP_SECP256R1_ENABLED` (same for the tests in the previous commit, which I missed earlier).

Reply:
Done